chipy_org/settings_test.py

```diff
@@ -3,7 +3,7 @@
 
 DATABASES = {"default": {"ENGINE": "django.db.backends.sqlite3", "NAME": ":memory:", "TEST": {}}}
 DEBUG = True
-ADMINS = "admin@chipy.org"
+ADMINS = ["admin@chipy.org"]
```

Comment:
Admins should always be plural, setting it with a string caused 10s of layers of failures in the test stack trace when I was trying to debug.